baxter_collaboration_lib/src/robot_interface/gripper.cpp

```diff
@@ -12,9 +12,15 @@ Gripper::Gripper(std::string _limb, bool _use_robot) :
 {
     if (not use_robot) return;
 
+    // create a publisher for the gripper's commands
     pub = rnh.advertise<EndEffectorCommand>(
                    "/robot/end_effector/" + _limb + "_gripper/command", 1);
 
+    // create a subscriber to the gripper's properties
```

Comment:
Fixed aesthetic bug: The properties subscriber is now created before the state subscriber. Because if the state subscriber is created first, it will do the first run calibration. But since the properties will not be known yet, there will be a `ROS WARN` about doing calibration on an `uninitialized` gripper type.